src/aviary/utils.py

```diff
@@ -79,6 +94,44 @@ def is_coroutine_callable(obj) -> bool:
     return False
 
 
+async def extract_answer_llm(
+    proposed: str,
+    options: list[str],
+) -> str | None:
+    """Extract the answer from a proposed answer and a list of options."""
+    if not proposed:
+        return None
+    for option in options:
+        if proposed.strip().casefold() == option.casefold().strip():
+            return option
+
+    try:
+        from litellm import acompletion
+    except ImportError as e:
+        raise ImportError(
+            "eval_answer requires the 'llm' extra for 'litellm'. Please:"
+            " `pip install aviary[llm]`."
+        ) from e
```

Comment:
Can you look at how `ToolSelector.__init__` takes `acompletion: "Callable[..., Awaitable[ModelResponse]] | None" = None`, and apply it as an input arg here?

What this does is let people use local models.

Also, feel free to YAGNI on this one